pkg/products/grafana/reconciler.go

```diff
@@ -127,59 +100,13 @@ func (r *Reconciler) Reconcile(ctx context.Context, installation *integreatlyv1a
 		return phase, nil
 	}
 
-	phase, err = r.ReconcileNamespace(ctx, operatorNamespace, installation, client, r.log)
-	if err != nil || phase != integreatlyv1alpha1.PhaseCompleted {
-		events.HandleError(r.recorder, installation, phase, fmt.Sprintf("Failed to reconcile %s ns", operatorNamespace), err)
-		return phase, err
-	}
-
-	phase, err = resources.ReconcileLimitRange(ctx, client, operatorNamespace, resources.DefaultLimitRangeParams)
-	if err != nil || phase != integreatlyv1alpha1.PhaseCompleted {
-		events.HandleError(r.recorder, installation, phase, fmt.Sprintf("Failed to reconcile LimitRange for Namespace %s", operatorNamespace), err)
-		return phase, err
-	}
-
-	phase, err = r.reconcileSecrets(ctx, client, installation)
```

Comment:
why are we doing it in bootstrap now instead of here?

Reply:
done. Thank you